src/OpenLoco/Windows/Main.cpp

```diff
@@ -0,0 +1,67 @@
+#include "../Graphics/Gfx.h"
+#include "../Interop/Interop.hpp"
+#include "../Map/Tile.h"
+#include "../Ui/WindowManager.h"
+#include "../ViewportManager.h"
+
+using namespace OpenLoco::Interop;
+
+namespace OpenLoco::Ui::Windows::Main
+{
+    static loco_global<int32_t, 0x00E3F0B8> gCurrentRotation;
```

Comment:
You can get rid of this loco_global by using `viewport::getRotation` instead.

Reply:
Yeah something for the future I think